src/common/math/Constant.h

```diff
@@ -0,0 +1,61 @@
+// Generated with tools/math-constant/math-constant.py
+// Do not modify!
+
+#ifndef COMMON_CONSTANT_H_
```

Comment:
lacks "math" component

Reply:
I did cargo cult from `src/common/math/Vector.h` that also lacks the `MATH` component, now fixed for `constant.h`.